src/IROperator.h

```diff
@@ -1636,6 +1636,39 @@ inline Expr count_trailing_zeros(Expr x) {
                                 {x}, Internal::Call::PureIntrinsic);
 }
 
+/** Divide two integers, rounding towards zero. This is the typical
+ * behavior of most hardware architectures, which differs from
+ * Halide's division operator, which is Euclidean (rounds towards
+ * -infinity). */
+inline Expr div_round_to_zero(Expr x, Expr y) {
+    Internal::match_types(x, y);
+    if (x.type().is_uint()) {
+        return x / y;
+    }
+    user_assert(x.type().is_int()) << "div_round_to_zero only takes integer arguments\n";
+    user_assert(y.type().is_int()) << "div_round_to_zero only takes integer arguments\n";
```

Comment:
I just copy-pasted those two lines from a similar IROperator function. I assumed it was because the condition would be visible to the user, which would indicate which operand (x or y) was not an integer.